internal/pkg/api/container/container.go

```diff
@@ -404,6 +404,69 @@ func ContainerShow(csp *wwapiv1.ContainerShowParameter) (response *wwapiv1.Conta
 	return
 }
 
+func ContainerRename(crp *wwapiv1.ContainerRenameParameter) (err error) {
+	// rename the container source folder
+	sourceDir := container.SourceDir(crp.ContainerName)
+	destDir := container.SourceDir(crp.TargetName)
+	err = os.Rename(sourceDir, destDir)
```

Comment:
I think we should probably also either remove the built containers from the provision dir or move them, too.